trunk/SUAVE/Analyses/Mission/Segments/Conditions/Aerodynamics.py

```diff
@@ -121,3 +121,6 @@ def __defaults__(self):
         
         # weights conditions
         self.weights.vehicle_mass_rate     = ones_1col * 0
+
+        # number of vortices (for AVL)
+        self.vortex_number = 0.0  # defined a number of vortices per meter with units [m^-1]
```

Comment:
Why is this here and not in the AVL analysis class?

Reply:
I removed this. Issue resolved.